__tests__/integration/verifyTemplateTitle.test.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import pupExpect from 'expect-puppeteer'
+import { testUserLogin } from './loginHelper'
+
```

Comment:
For the record, Aaron write this, I did not.  Note that this is called an "integration" test, but it uses fixture data and is focused on a simple thing.

What was broken had to do with a mismatch between was going in redux vs. what was expected from redux.  I would be keen to learn techniques to make our unit tests less vulnerable to this sort of problem.